utils/setupTestEnvLocal

```diff
@@ -138,4 +139,5 @@ mv version_temp.yaml version.yaml
 ./utils/createConfigFileForTesting
 mkdir temp
 cp config.yaml temp/config.yaml
-rm config.yaml
\ No newline at end of file
+rm config.yaml
+touch .testEnvRunning
```

Comment:
Where is this file being used?

Reply:
The `.testEnvRunning` file?

Reply:
Yes.

Reply:
`.testEnvRunning` is placed into the installation directory once all of the necessary files and directories have been copied there in order to begin testing. The file is removed in `cleanTestEnvLocal`/`cleanTestEnvCicd` when the testing environment has been destroyed. I added a check for this file in supertokens/supertokens-core#429 that displays an error message instead of running the test suite when this file does not exist. I plan to add a check for this file in `startTestingEnv` so that the testing environment will display an error message instead of running twice and producing undefined behavior.

I also added this file so that I could test for it in a new script that I am writing, `runCore`. This script runs `./startTestingEnv --wait`, waits for `.testEnvRunning` to appear, and then runs the core. When `runCore` is terminated, it also kills the `startTestingEnv` child process.

Reply:
Ah okay! Makes sense.